### PR TITLE
I#69 respond to group invite

### PR DESCRIPTION
**Pull request by ryanknauer**

### CODE REVIEW

File: api-customer/functions/group-invite-response/index.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        const game_group_invite_id = event.pathParameters.game_group_invite_id;
+        const invite_status = body.invite_status;
+
+        var client = yield pool.connect();
+        try {
+
+            const query = SQL `
+                UPDATE game_group_invite SET
+                invite_status_id = ${invite_status}
+                WHERE id = ${game_group_invite_id};
```
Comment (anmolg) [inline api-customer/functions/group-invite-response/index.js]: just use event.pathParameters etc. directly in this

similarly for invite status.

File: api-customer/functions/group-invite-response/index.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    console.log(event);
+    console.log(context);
+
+    context.callbackWaitsForEmptyEventLoop = false;
+
+    let body = event.body;
+    try {
+        body = JSON.parse(event.body);
+    } catch (err) {}
+
```
Comment (shreyansjain) [inline api-customer/functions/group-invite-response/index.js]: Quick error validation here to check if `body` is empty or not

File: api-customer/functions/group-invite-response/index.js [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    Promise.coroutine(function* () {   
+
+        var client = yield pool.connect();
+        try {
+
+            const query = SQL `
+                UPDATE game_group_invite SET
```
Comment (shreyansjain) [inline api-customer/functions/group-invite-response/index.js]: Can you reformat the query: something like: 
```
UPDATE    game_group_invite 
SET        invite_status_id = ${event.pathParameters.game_group_invite_id}
WHERE      id = ${body.invite_status};
```

This is usually Anmol's preference (and now mine too)

File: api-customer/functions/group-invite-response/index.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+        try {
+
+            const query = SQL `
+                UPDATE  game_group_invite 
+                SET     invite_status_id = ${event.pathParameters.game_group_invite_id}
+                WHERE   id = ${body.invite_status};
+                `; 
+
+            yield client.query('BEGIN');
+            let result = yield client.query(query);
```
Comment (anmolg) [inline api-customer/functions/group-invite-response/index.js]: you can remove let result. Nothing is happening with it, so just yielding is necessary

File: api-customer/functions/group-invite-response/index.js [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+            yield client.query('COMMIT');
+
+            let response = {
+                statusCode: 200
+            };
+            callback(null, response);
```
Comment (anmolg) [inline api-customer/functions/group-invite-response/index.js]: console.log before doing a callback

File: api-customer/functions/group-invite-response/index.js (hunk 4 above)
Comment (anmolg) [inline api-customer/functions/group-invite-response/index.js]: Can you remove "let result =" since it is redundant?

**Pull request merged**